test/integration/challenges.test.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const authService = require('../../services/authService')
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+
+const config = require('config')
+const cookieName = config.get('userToken.cookieName')
+
+chai.use(chaiHttp)
+
+describe('Challenges', function () {
+  let jwt
+
+  beforeEach(async function () {
+    const userId = await addUser()
+    jwt = authService.generateAuthToken({ userId })
+  })
+
+  afterEach(async function () {
+    await cleanDb()
+  })
+
+  describe('GET /challenges', function () {
+    it('Should return the available challenges', function (done) {
+      chai
+        .request(app)
+        .get('/challenges')
+        .set('cookie', `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err)
+          }
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.be.equal('Challenges returned successfully!')
+          expect(res.body.challenges).to.be.a('array')
```

Comment:
@Ishikabaid you need to add a dummy challenge here since DB is empty. You can refer [this](https://github.com/Real-Dev-Squad/website-backend/blob/1fbda36c6b1cc764a2b099d1431d5160c681b882/test/integration/tasks.test.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L198)